nextcord/__init__.py

```diff
@@ -13,7 +13,7 @@
 __author__ = 'tag-epic & Rapptz'
 __license__ = 'MIT'
 __copyright__ = 'Copyright 2015-present Rapptz'
-__version__ = '2.0.0a3'
+__version__ = '2.0.0a4'
```

Comment:
the versionadded was below ``components`` as that was added in 2.0, could you move it like this?